backend/src/schema/resolvers/users.js

```diff
@@ -57,6 +57,10 @@ export default {
     },
   },
   Mutation: {
+    acceptTermsAndCondition: async (object, args, context, resolveInfo) => {
```

Comment:
> [<img alt="roschaefer" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/roschaefer) **Authored by [roschaefer](https://github.com/roschaefer)**
_<time datetime="2019-08-16T08:36:56Z" title="Friday, August 16th 2019, 10:36:56 am +02:00">Aug 16, 2019</time>_

---
> **Outdated (history rewrite)** - original diff
---
```diff
@@ -57,6 +57,10 @@ export default {
     },
   },
   Mutation: {
+    acceptTermsAndCondition: async (object, args, context, resolveInfo) => {
```


Remove this code